### PR TITLE
Update configs

### PR DESCRIPTION
**Pull request by igorechek06**
Added plugin and core installer
Deleted easywhitelist config
Added shebang to start.sh
Added .gitigonre

### CODE REVIEW

**Review by Calamity34 — APPROVED** [review]
lgtm

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,14 @@
+*
+
+!plugins/
```
Comment (arslee07) [inline .gitignore:3]: please exclude `plugins/CoreProtect/database.db`

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python
```
Comment (arslee07) [inline setup.py:1]: maybe rewrite to bash, not python?
you can iterate through lines of the file easily:

```bash
cat plugins.txt | while read line || [[ -n $line ]];
do
   # do something with $line here
done
```

File: plugins.txt [hunk 3]
```diff
@@ -0,0 +1,13 @@
+https://nc.igorek.dev/s/K7XMYQS9DJk2EF7/download?path=%2F&files=AuthMe.jar@AuthMe.jar
```
Comment (arslee07) [inline plugins.txt:1]: just a quick question: should we updates the plugins links if we will update them on the nextcloud?

File: .gitignore (hunk 1 above)
Comment (arslee07) [reply .gitignore:3]: ...and `plugins/Pl3xMap/data`

**Review by arslee07 — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (arslee07) [reply .gitignore:3]: ...and `plugins/FastLogin/FastLogin.db`

Comment (arslee07) [reply .gitignore:3]: ...and `plugins/DeadChest/chestData.yml` with `plugins/DeadChest/deadchest.log`

Comment (arslee07) [reply .gitignore:3]: alright, just exclude `*.log`

File: plugins.txt (hunk 3 above)
Comment (igorechek06) [reply plugins.txt:1]: Yes!

File: setup.py (hunk 2 above)
Comment (igorechek06) [reply setup.py:1]: I tried but it didn't work

File: plugins.txt (hunk 3 above)
Comment (arslee07) [reply plugins.txt:1]: ok, i got it

**Review by arslee07 — APPROVED** [review]

**Pull request merged**